daemon/containerd/mount.go

```diff
@@ -38,12 +47,14 @@ func (i *ImageService) Mount(ctx context.Context, container *container.Container
 func (i *ImageService) Unmount(ctx context.Context, container *container.Container) error {
 	root := container.BaseFS
 
-	if err := mount.UnmountAll(root, 0); err != nil {
-		return fmt.Errorf("failed to unmount %s: %w", root, err)
-	}
+	if strings.HasSuffix(root, rootfsMountSuffix) {
```

Comment:
In this case we don't unmount, but still reset `container.BaseFS`; should this be an error condition? (Should it be logged?)